reV/bespoke/bespoke.py

```diff
@@ -796,17 +928,28 @@ def __init__(self, excl_fpath, res_fpath, tm_dset,
                 - n_turbines: the number of turbines
                 - system_capacity: wind plant capacity
                 - aep: annual energy production
+                - fixed_charge_rate: user input fixed_charge_rate if included
+                  as part of the sam system config.
                 - self.wind_plant: the SAM wind plant object, through which
                 all SAM variables can be accessed
-                - cost: the annual cost of the wind plant (from cost_function)
-        cost_function : str
-            The cost function as a string, should return the annual cost
-            of the wind farm. Variables available are:
-                - n_turbines: the number of turbines
-                - system_capacity: wind plant capacity
-                - aep: annual energy production
-                - self.wind_plant: the SAM wind plant object, through which
-                all SAM variables can be accessed
+                - capital_cost: plant capital cost as evaluated
+                  by `capital_cost_function`
+                - fixed_operating_cost: plant fixed annual operating cost as
+                  evaluated by `fixed_operating_cost_function`
+                - variable_operating_cost: plant variable annual operating cost
+                  as evaluated by `variable_operating_cost_function`
+        capital_cost_function : str
+            The plant capital cost function as a string, must return the total
+            capital cost in $. Has access to the same variables as the
+            objective_function.
+        fixed_operating_cost_function : str
+            The plant annual fixed operating cost function as a string, must
+            return the fixed operating cost in $/year. Has access to the same
+            variables as the objective_function.
+        variable_operating_cost_function : str
+            The plant annual variable operating cost function as a string, must
+            return the variable operating cost in $/kWh. Has access to the same
+            variables as the objective_function.
```

Comment:
Excellent docstrings. As an end user, I am especially happy about the unit specifications.